packages/diagrams/lib/src/text.dart

```diff
@@ -0,0 +1,204 @@
+// Copyright 2019 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:diagram_capture/diagram_capture.dart';
+import 'package:flutter/material.dart';
+import 'diagram_step.dart';
+
+class TextDiagram extends StatelessWidget implements DiagramMetadata {
+  const TextDiagram(this.name);
+
+  @override
+  final String name;
+
+  @override
+  Widget build(BuildContext context) {
+    final TextPainter textPainter = TextPainter(
+      text: const TextSpan(
+        text:' Aalg我हिन्दी ',
+        style: TextStyle(
+          fontSize: 100,
+          color: Colors.black,
+        ),
+      ),
+      textAlign: TextAlign.left,
+      textDirection: TextDirection.ltr,
+    );
+    switch (name) {
```

Comment:
Why this switch?

Reply:
This allows for adding additional diagrams using the same the same class in the future based off of a different name. It is not being used currently, so I can remove this for now if you want.

Reply:
I'd just remove it and add that when we actually need it.